packages/taquito/test/contract/rpc-contract-provider.spec.ts

```diff
@@ -161,6 +163,15 @@ describe('RpcContractProvider test', () => {
     });
   });
 
+  describe('BigMapAbstraction', () => {
+    it('returns undefined on bad key in BigMap', async done => {
```

Comment:
Do you think it would be interesting to be more explicit with regards to the error that is thrown by rpcContractProvider.getBigMapKeyByID?

For instance, we could have a test for HTTPResponseError where we mock the response using:
`rpcContractProvider.getBigMapKeyByID.mockRejectedValue(new HTTPResponseError(Status.NotFound))`
In this case we would expect undefined

Another test could be mocking the value using:
`rpcContractProvider.getBigMapKeyByID.mockRejectedValue(new Error())`
In this case we would expect the call to throw this error again